components/button/index.css

```diff
@@ -91,68 +89,39 @@ governing permissions and limitations under the License.
 
 }
 
-
 .spectrum-Button {
   @inherit: %spectrum-BaseButton;
   @inherit: %spectrum-ButtonWithFocusRing;
 
-
   border-radius: var(--mod-button-border-radius, var(--spectrum-button-border-radius));
   border-width: var(--mod-button-border-width, var(--spectrum-button-border-width));
   border-style: solid;
-  block-size: auto;
+  font-size: var(--mod-button-font-size, var(--spectrum-button-font-size));
+  font-weight: var(--mod-bold-font-weight, var(--spectrum-bold-font-weight));
+  gap: var(--spectrum-button-padding-label-to-icon);
   height: var(--mod-button-height, var(--spectrum-button-height));
-
-  min-width: var(--mod-button-min-width, var(--spectrum-button-min-width));
-  min-block-size: var(--mod-button-min-width, var(--spectrum-button-min-width));
-  position: relative;
-
-
-  padding-block-start: 0;
-  padding-block-end: 0;
```

Comment:
Lots of stuff over here is either duplicated or has not active value. Much of it was likely removed at tooling time, but easier to understand what's going on when they're not here.